indico/web/client/styles/partials/_protection_indicator.scss

```diff
@@ -0,0 +1,60 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'partials/icons';
+@use 'design_system';
+
+#protection-indicator {
```

Comment:
With`align-items: center` this looks much better since the icon is nicely centered:

old:
![image](https://github.com/indico/indico/assets/179599/0bc6a46d-4723-4115-a643-8ceb25009f70)
new:
![image](https://github.com/indico/indico/assets/179599/32c922e9-31b3-4f4a-9eef-730fe366fe6b)